src/lib.rs

```diff
@@ -260,9 +374,94 @@ impl<G: Game> IsmctsHandler<G> {
     pub fn state(&self) -> &G {
         &self.root_state
     }
+
+    pub fn dotty_graph(&self) {
+        println!("{}", self.dotty_graph_string());
+    }
+
+    pub fn dotty_graph_string(&self) -> String {
```

Comment:
I know there are other methods that write to console, but doing that for the graph seems unnecessary since it is going to be rendered from the file on disk.
```suggestion
    #[must_use]
    pub fn dotty_graph(&self) -> String {
```